runbotics/runbotics-actions-windows/src/automation/office/excelErrorMessage.ts

```diff
@@ -4,98 +4,110 @@ import { Injectable } from '@nestjs/common';
+    static deleteColumnsIncorrectInput(): string {
         return `
-            ${this.getDisclaimer(e)}
-            Input must be range or Array (e.g. "A1:C3" or ["A1", "B2", "E5"]).
-            Check targetCells in Input tab above.
-            Try to pass it as variable (e.g. #{myArray}).
+            Input must be range or Array (e.g. "A", "F:J" or ["A", "B", "C"]).
+            ${this.getCheckInputTip()}
         `;
     }
 
-    static deleteColumnsIncorrectInput(e?: Error): string {
+    static createWorksheetIncorrectInput(): string {
         return `
-            ${this.getDisclaimer(e)}
-            Input must be range or Array (e.g. "A", "F:J" or ["A", "B", "C"]).
-            Check columnRange in Input tab above.
-            Try to pass it as variable (e.g. #{myArray}).
-        `;
+            Worksheet name is incorrect or already taken.
+            ${this.getCheckInputTip()}
+        `
     }
 
-    static worksheetIncorrectInput(shouldExist: boolean): string {
-        return shouldExist
-            ? "Worksheet doesn't exist."
-            : "Worksheet name incorrect or already taken."
-        ;
+    static insertColumnsIncorrectInput(): string {
+        return `
+            Column has to be a column letter or number (e.g. "C").
+            Amount has to be a number (eg. 5).
+            ${this.getCheckInputTip()}
+            `
     }
 
-    static insertColumnsIncorrectInput(e?: Error): string {
+    static insertRowsIncorrectInput(): string {
         return `
-            ${this.getDisclaimer(e)}
-            Column has to be a column letter or number, e.g. "C" or 3.
-            Amount has to be a whole positive number, eg. 5.
+            Row has to be a whole positive number (e.g. 5).
+            Number has to be a number (e.g. 5).
+            ${this.getCheckInputTip()}
         `
     }
 
-    static insertRowsIncorrectInput(e?: Error): string {
+    /* Incorrect utils input */
+
+    static divideCellCoordinatesIncorrectInput(): string {
         return `
-            ${this.getDisclaimer(e)}
-            Row has to be a whole positive number, e.g. 5.
-            Number has to be a whole positive number, eg. 5.
+            Start/End cell must be a string (e.g. "A1").
+            ${this.getCheckInputTip()}
         `
     }
 
-    /* Other */
-    private static getDisclaimer(e?: Error): string {
+    static worksheetNameAlreadyTakenIncorrectInput(): string {
         return `
-                ${e ?? ''}
-                Common (related to action, not error) solutions:
-            `
+            Worksheet name is already taken.
+            ${this.getCheckInputTip()}
+        `;
+    }
+
+    static worksheetNameDoesNotExistIncorrectInput(): string {
+        return `
+            Worksheet name doesn't exist.
+            ${this.getCheckInputTip()}
+        `;
+    }
+
+
+    static getColumnCoordinateIncorrectInput(): string {
+        return `
+            Column must be a string (e.g. "A").
```

Comment:
why do we get an error once that says the column must be a number or a letter (insertColumnsIncorrectInput) and once just a letter?